rotkehlchen/tests/unit/test_chains_aggregator.py

```diff
@@ -327,3 +334,24 @@ def test_detect_evm_accounts_spam_tx_gnosis(gnosis_manager: 'GnosisManager') ->
         assert gnosis_manager.transactions.address_has_been_spammed(evm_address) is True
         # verify that the gnosiscan API get's queried with the last recorded blocknumber
         assert all(f'startBlock={block_number}' in call.args[0] for call in mocked_get.mock_calls), "URL must contain 'startBlock=' and correct block number"  # noqa: E501
+
+
+@pytest.mark.vcr(filter_query_parameters=['apikey'])
+@pytest.mark.freeze_time(datetime.datetime.fromtimestamp(1717116305))  # noqa: DTZ006
+@pytest.mark.parametrize('base_accounts', [['0xeA2B3D309bC480Fe385BBF8aEF6D45D81825A784']])
+def test_detect_spammed_transaction_new_token(
+        base_manager: 'BaseManager',
+        base_accounts: list[ChecksumEvmAddress],
+) -> None:
+    """Check that processing spam transactions for tokens that haven't
+    been tracked before detects better spam. The address has only one transaction in the range
+    for a non tracked token that is only doing spam transfers.
```

Comment:
Also improve docstring: In the range? What range? It does not make sense. Can you please ping with the improved docstring in discord before pushing?

```suggestion
    been tracked before detects spam better. The address has only one transaction in the range
    for a non tracked token that is only doing spam transfers.
```